projects/projects-model.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../data/dbConfig.js')
+
+module.exports = {
+    getProjects,
+    findById,
+    addProject,
+    addAction,
+    findActions
+  };
+
+  function getProjects() {
+    return db('projects')
+}
+
+function findById(id) {
+    return db('projects')
+        .where({ id })
+        .first()
+        .then(user => {
+            if(user){
+                return user
+            } else {
+              return null;
+            }
+        })
+}
+
+function addProject(project) {
+    return db('projects')
+     .insert(project)  
+  }
+
+  function addAction(actionData, project_id){
+    return db('actions')
+        .insert({...actionData, project_id})   
+}
+
+function findActions(id) {
```

Comment:
Great work using the spread operator in `addAction`. Even more impressive is this `findActions` function for the many to many relationship case! Though you're not taking FULL advantage of it. See, now that you have a nicely joined table you can also grab the `project_name` column for a more descriptive function return. ie. `.select('action_description', 'action_notes')`